src/translations/en.json

```diff
@@ -83,6 +83,8 @@
     "Limit": "Limit",
     "load": "load",
     "Load": "Load",
+    "MapManualRefresh": "Manual update of geographical view\u00a0:",
```

Comment:
There is no space before colons in english.

Reply:
✓